### ISSUE TITLE
Visual Testing: Replace the current run script to use `concurrently`

### ISSUE THREAD

**Issue by AlexVelezLl** [learningequality/kolibri-design-system#676]
## Description

Currently for the visual testing project, we need to run the dev server and percy to be able to run the tests. At the moment to do this, we are calling a script `runVisualTests.js` that sets up the server and runs the tests once the server is ready, but this is an approach that can be replaced with more suitable tools such as concurrently.

**Comment by KshitijThareja** [comment]
Thanks @AlexVelezLl for creating the follow-up issue. Will have the PR up soon :+1:

**Comment by AlexVelezLl** [comment]
Closed by #677
